libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -792,8 +792,8 @@ void EthStratumClient::processReponse(Json::Value& responseObject)
 
 					cnote << "Subscribed to stratum server";
 
-					m_nextWorkDifficulty = 1;
-					
+					m_nextWorkBoundary = h256("0xffff000000000000000000000000000000000000000000000000000000000000");
```

Comment:
Why not all `f`?

Reply:
This seems to be the max target value used in all ethereum code. Not sure why it's not all F's.